packages/teleport-project-generator-preact/src/index.ts

```diff
@@ -49,6 +47,21 @@ export const createPreactProjectGenerator = () => {
       generator: htmlFileGenerator,
       path: ['src'],
       fileName: 'index',
+      options: {
+        appRootOverride: customBodyContent(),
+        customHeadContent: customHeadContent(),
+        customScriptTags: [
+          {
+            target: 'body',
+            type: 'defer',
+            path: `<%= htmlWebpackPlugin.files.chunks['bundle'].entry %>`,
```

Comment:
isn't this the root actually? and this:
```
<%= htmlWebpackPlugin.options.ssr({
	url: '/'
}) %>
```

would be another custom script tag?

Reply:
Yes, the SSR tag is outside the script and this is added inside a script tags